api/app/dao/date_util.py

```diff
@@ -2,14 +2,14 @@
 
 import pytz
 
-from app.utils import convert_bst_to_utc
+from app.utils import convert_aest_to_utc
 
 
 def get_months_for_financial_year(year):
     return [
-        convert_bst_to_utc(month) for month in (
-            get_months_for_year(4, 13, year) +
-            get_months_for_year(1, 4, year + 1)
+        convert_aest_to_utc(month) for month in (
+            get_months_for_year(7, 13, year) +
+            get_months_for_year(1, 7, year + 1)
```

Comment:
As above regarding financial year date ranges.

Maybe this and the other method can be moved into `utils/notification_utils`